client/models.py

```diff
@@ -1,9 +1,11 @@
 from django.db import models
 from django.core.validators import MinLengthValidator
-from buy_together_app.models import User
+from buy_together_app.models import CustomUser
+from django.contrib.auth.models import User
 
 
-class Client(User):
+class Client(CustomUser):
+    client_account = models.OneToOneField(User, null=True, on_delete=models.CASCADE)
```

Comment:
why client account can be null?

Reply:
Its a one to one relation it cant be full on both sides